methods/secular/Secular_Requirement_Validation.ipynb

```diff
@@ -9,7 +9,8 @@
     "**Original code authored by:** David Bekaert, Heresh Fattahi, Eric Fielding, and Zhang Yunjun \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    cor_vel = f'{mintpy_dir}/IONO_velocity.h5'
</pre><p>This should be <code>tropo_velocity.h5</code> or something similar. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>